app/server/api/search_api.py

```diff
@@ -52,13 +116,28 @@ def get(self):
         if search_string == '':
             if search_type == 'transfer_accounts':
                 final_query = TransferAccount.query.filter(TransferAccount.is_ghost != True)\
-                    .execution_options(show_all=True)
+                    .execution_options(show_all=True)\
+                    .join(User, User.default_transfer_account_id == TransferAccount.id)
+                if sort_by_arg == 'rank':
+                    # There's no search rank when there's no query string, so do chrono instead
+                    final_query = final_query.order_by(order(User.created))                    
+                else:
+                    final_query = final_query.order_by(order(sort_by))
+
                 transfer_accounts, total_items, total_pages = paginate_query(final_query, TransferAccount)
                 result = transfer_accounts_schema.dump(transfer_accounts)
                 data = { 'transfer_accounts': result.data }
             else:
                 final_query = CreditTransfer.query.filter()\
-                    .execution_options(show_all=True)
+                    .execution_options(show_all=True)\
```

Comment:
Note: The show_all is going to be nuked when the proper search branch is merged and the merge conflict fun begins